native/connector/connect-and-monitor.sh

```diff
@@ -6,11 +6,15 @@ virtmicPortFlId=$1
 virtmicPortFrId=$2
 targetNodeSerial=$3
 
+source $PROJECT_ROOT/connector/util.sh
+
+exec 2>>`UtilGetLogPathForFile $(basename $0)`
```

Comment:
Why only stderr?

Reply:
because STDIN and STDOUT are used to communicate with the browser